### PR TITLE
Update PR template to allow new PRs

### PR DESCRIPTION
**Pull request by alexandrebodin**
#### Description of what you did:

Update PR template to allow new PRs

#### My PR is a:
- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [X] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:
- [ ] Admin
- [X] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:
- [X] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,42 +1,35 @@
...
-
 <!--
 Hello 👋 Thank you for submitting a pull request.
 
-To help us to merge your PR please follow these bullet points:
-- You wrote documentation
-- You wrote tests
-- Refer to the issue you are closing in your PR description - fix #issue
-- Let us know if this PR is WIP or ready to merge
+To help us merge your PR, make sure to follow the instructions below:
```
Comment (davidkartuzinski) [inline .github/PULL_REQUEST_TEMPLATE.md]: **these** instructions below

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,42 +1,35 @@
...
+- Create or update the documentation.
+- Create or update the tests.
+- Refer to the issue you are closing in the PR description - fix #issue
+- Specify if the PR is in WIP (work in progress) state or ready to be merged
```
Comment (davidkartuzinski) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do we want to say: Specify in the title, if the PR is in a WIP (word in progrss) state or ready to be merged, e.g. title: [WIP] Update the docs

**Comment by davidkartuzinski** [comment]
I made a few comments...

**Comment by alexandrebodin** [comment]
the PR is already merged if you want to change things you will have to make a new PR

**Pull request merged**